project/AppDependencies.scala

```diff
@@ -2,25 +2,20 @@ import sbt.*
 
 object AppDependencies {
 
-  private val pactVersion = "4.4.0"
-
   private lazy val bootstrapPlayVersion = "8.5.0"
   private lazy val hmrcMongoVersion     = "1.9.0"
 
   private val compile: Seq[ModuleID] = Seq(
```

Comment:
some dependencies which can possibly be bumped, maybe just recently newer releases. See if we can bump them up?